### ISSUE TITLE
Reactions plugin doesn't use configured maximum

### ISSUE THREAD

**Issue by Nixinova** [lowlighter/metrics#376]
The reactions plugin maximum config value doesn't do anything. [These metrics](https://raw.githubusercontent.com/Nixinova/Nixinova/0c0c34cc27f4a9844c2db4df65d320434c86fbb5/github-metrics.svg) ignored the configured maximum which was definitely read by metrics ([workflow](https://github.com/Nixinova/Nixinova/runs/2786732937?check_suite_focus=true) line 405) and loaded all comments instead of the set 200.

**Comment by lowlighter** [comment]
It's because this option was split after #333

Previously, issues and prs were fetched first so it was possible that your comments were actually never fecthed (problem being that GitHub considers the "initial comment" of an issue/pr different from subsequent comments), so it was decided to split it into 2 options:

```yaml
    plugin_reactions_limit: 200                 # Compute reactions over last 200 issue comments
    plugin_reactions_limit_issues: 100          # Compute reactions over laste 100 issues/pull requests opened
```
The +67 comments you have are probably from the second option (i.e. issues and prs you opened)
